app/Http/Controllers/Api/AuthController.php

```diff
@@ -44,7 +44,7 @@ public function login(Request $request)
         }
 
         $credentials = [
-            'username' => $request->get('username'),
+            'email' => $request->get('email'),
```

Comment:
Same applies to this please @saidabdul80

Reply:
Noted